client/state/selectors/test/get-current-route-parameterized.js

```diff
@@ -0,0 +1,67 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+
+/**
+ * Internal dependencies
+ */
+import getCurrentRouteParameterized from 'state/selectors/get-current-route-parameterized';
+
+describe( 'getCurrentRouteParameterized()', () => {
+	const ui = {
+		route: {
+			path: {
+				current: '/test/url/testsite.blog',
+			},
+		},
+	};
+
+	const sites = {
+		items: { '12345': { URL: 'http://testsite.blog' } },
+	};
+
+	test( 'it returns null when state is missing the path', () => {
+		const state = {
+			ui: {},
+			sites: sites,
+		};
+
+		expect( getCurrentRouteParameterized( state, 12345 ) ).to.be.null;
+	} );
+
+	test( 'it returns null when state is missing the site', () => {
+		const state = {
+			ui: ui,
+			sites: { items: {} },
+		};
+
+		expect( getCurrentRouteParameterized( state, 12345 ) ).to.be.null;
+	} );
+
+	test( 'it replaces the site slug with :site', () => {
+		const state = {
+			ui: ui,
+			sites: sites,
+		};
```

Comment:
Super minor but we don't have to repeat props here and in other parts of this test case:

```javascript
const state = {
  ui,
  sites,
};
```